Add log rotation information (#2121)
hashicorp/terraform-website: content/enterprise/admin/infrastructure/logging.mdx

For a Standalone installation of Terraform Enterprise:

```sh
replicatedctl app-config set log_forwarding_config --value "$(cat fluent-bit.conf)"
```

For an Active/Active installation of Terraform Enterprise:

```sh
tfe-admin app-config -k log_forwarding_config -v "$(cat fluent-bit.conf)"
```

Once configured, the Terraform Enterprise application settings show the
`log_forwarding_config` setting in escaped JSON string format:

```json
    "log_forwarding_config": {
        "value": "# Match all logs and do not forward them anywhere.\n[OUTPUT]\n    Name null\n    Match *\n"
    },
```

> Here's what that escaped JSON string will render to:

Reviewer: ```suggestion
That escaped JSON string renders to the following:
```

filtering the audit logs based on the `[Audit Log]` string, and forwarding just
the audit logs to the desired destination.

## Log Rotation

Log forwarding uses the `journald` Docker logging driver to send Terraform
Enterprise logs to `systemd-journald`. This can cause increased disk utilization
for the `/var/log/journal` path.

> To limit this disk utilization, configure the `SystemMaxFileSize` and

Reviewer: Beautifully done!  Two thoughts:
- What do we mean by "reasonable values"? I wonder if we could say something more specific OR simplify to say "To limit disk use, set the `SystemMaxFileSize` and `SystemMaxFiles` settings within `etc/systemd/journald.conf`.
- I would move the explanation of what the example does to before the example. So I'd say "The following example tells..."   I think it's nice to tell folks what they're about to see before they see it.

Author: I like it! Thank you so much!
